docs/quick-start.md

```diff
@@ -114,103 +114,8 @@ This configuration adds a new setting, `public_folder`. While `media_folder` spe
 >If `public_folder` is not set, Netlify CMS will default to the same value as `media_folder`, adding an opening `/` if one is not included.
 
 ### Collections
-Collections define the structure for the different content types on your static site. Since every site is different, the `collections` settings will be very different from one site to the next.
+Collections define the structure for the different content types on your static site. They will be the main part of your config, and are explained in detail on the [Collections](collections.md) page.
```

Comment:
@erquhart Is it good to drop everything except this from the quick-start page?

Reply:
Yeah, that's fine.